### ISSUE TITLE
Configuring underline

### ISSUE THREAD

**Issue by etogithub** [facebook/PathPicker#320]
Thanks for this great tool.

The default method of underlining looks pretty good (slightly shaded) in the sample pictures but in my terminal (alacritty+DejaVuSansMono+tmux) is bold and pretty hard on the eyes.

Are there any plans to make this underlining indicator configurable?

I was looking around in formattedText.py and if I were to modify it I might even just colorize the selectable files as opposed to underlining them.  Just an idea.

Thanks.

**Comment by pcottle** [comment]
Yeah I'd be supportive of this! No immediate plans on my end but feel free to tweak it. Mind attaching a picture?
